client/src/components/PodcastsIndex.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from "react";
+import PodcastTile from "./PodcastTile";
+
+const PodcastsIndex = (props) => {
+  const [podcasts, setPodcasts] = useState([]);
+
+  const getPodcasts = async () => {
+    try {
+      const response = await fetch("api/v1/podcasts");
+      if (!response.ok) {
+        throw new Error(`${response.status} (${response.statusText})`);
+      }
+      const body = await response.json();
+      setPodcasts(body.podcasts);
+    } catch (error) {
+      console.error(`Error in fetch: ${error.message}`);
+    }
+  };
+
+  useEffect(() => {
+    getPodcasts();
+  }, []);
+
+  const podcastTiles = podcasts.map((podcast, index) => {
+    return (
+      <div>
+        <PodcastTile key={index} podcast={podcast} />
+      </div>
```

Comment:
Any styling for a component should be created _on_ that component, so we should remove these `div`s here and if we want that styling it should be added to the PodcastTile component. 

Some other red flags here are that we are using the `index` for a `key`. Each podcast record should have its own `id` that we can use for the key property. Using the index is very risky and dangerous because it's using that objects index place from the array that we are mapping over, and the order of that array is subject to change if we add or remove any objects. If the order of the array is altered, those original `key` values are no longer accurate because their original `index` (place within the array) has now changed, which means that that special `key` property value has now also changed. They `key` property here is very important for React to be able to distinguish between each `PodcastTile` that we create, because as far as React is concerned they are all identical PodcastTile components